runtime/parachains/src/paras_inherent.rs

```diff
@@ -728,7 +727,7 @@ fn apply_weight_limit<T: Config + inclusion::Config>(
 	let (total, indices) = random_sel::<UncheckedSignedAvailabilityBitfield, _>(
 		&mut rng,
 		bitfields.clone(),
-		|_| bitfield_weight::<T>(),
+    	|_| <<T as Config>::WeightInfo as WeightInfo>::enter_bitfields(),
```

Comment:
This is called per bitfield, so I am not sure this is what you want, the ellipsized arg would be the bitfield to meadure

Reply:
This should be correct, we assume `<<T as Config>::WeightInfo as WeightInfo>::enter_bitfields()` will always return the weight of processing one bitfield. We don't need to know anything about the bitfield itself though